docs/core/install/localized-intellisense.md

```diff
@@ -0,0 +1,124 @@
+- [Visual Studio 2019 16.3](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019) or later versions.
+
+## Download and install the localized IntelliSense files
+
+> [!IMPORTANT]
+> This procedure requires that you have administrator permission to copy the IntelliSense files to the .NET Core installation folder.
+
+1. Go to the [Download IntelliSense files](https://dotnet.microsoft.com/download/dotnet-core) page.
+
+1. Download the IntelliSense file for the language and version you'd like to use.
+
+1. Extract the contents of the zip file.
+
+1. Navigate to the .NET Core installation folder. By default, it's under *C:\Program Files\dotnet\packs*.
+
+   - Choose which SDK you want to install the IntelliSense for and navigate to the associated path. You have the following options:
+
+      | SDK type        | Path                               |
+      | --------------- | ---------------------------------- |
+      | .NET Core       | *Microsoft.NETCore.App.Ref*        |
+      | Windows Desktop | *Microsoft.WindowsDesktop.App.Ref* |
+      | .NET Standard   | *NETStandard.Library.Ref*          |
+   
+   - Navigate to the version you want to install the localized IntelliSense for. For example, *3.1.0*.
+   - Open the *ref* folder.
+   - Open the moniker folder. For example, *netcoreapp3.1*.
+
+   So, the full path that you'd navigate to would look similar to *C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Ref\3.1.0\ref\netcoreapp3.1*.
+
+1. Create a subfolder inside the moniker folder you just opened. The name of the folder indicates which language you want to use. The following table specifies the different options:
+
+   | Language              | Folder name |
+   | --------------------- | ----------- |
+   | Chinese (simplified)  | *zh-hans*   |
+   | Chinese (traditional) | *zh-hant*   |
+   | French                | *fr*        |
+   | German                | *de*        |
+   | Italian               | *it*        |
+   | Japanese              | *ja*        |
+   | Korean                | *ko*        |
+   | Brazilian Portuguese  | *pt-br*     |
+   | Russian               | *ru*        |
+   | Spanish               | *es*        |
+
+1. Copy the *.xml* files you extracted on step 3 to this new folder. The *.xml* files are broken down by SDK folders, so copy them to the matching SDK you chose on step 4.
+
+## Modify Visual Studio language
+
+For Visual Studio to use a different language for IntelliSense, it requires you to install the language pack you want to use. This can be done [during installation](/visualstudio/install/install-visual-studio#step-6---install-language-packs-optional) or at a later time by modifying the Visual Studio installation. If you already have Visual Studio configured to the language of your choice, your IntelliSense installation is ready.
```

Comment:
```suggestion
For Visual Studio to use a different language for IntelliSense, install the appropriate language pack. This can be done [during installation](/visualstudio/install/install-visual-studio#step-6---install-language-packs-optional) or at a later time by modifying the Visual Studio installation. If you already have Visual Studio configured to the language of your choice, your IntelliSense installation is ready.
```